hive-metastore/src/main/java/org/apache/iceberg/hive/HiveViewOperations.java

```diff
@@ -175,11 +172,14 @@ public void doCommit(ViewMetadata base, ViewMetadata metadata) {
                 .filter(key -> !metadata.properties().containsKey(key))
                 .collect(Collectors.toSet());
       }
-
-      setHmsTableParameters(newMetadataLocation, tbl, metadata, removedProps);
-
+      HMSTablePropertyHelper.setHmsTableParameters(
```

Comment:
Suggested an update for the name, so this line should change too
```suggestion
      HMSTablePropertyHelper. updateHmsTableForIcebergView(
```

Reply:
Fixed.